app/browser-sync.ts

```diff
@@ -93,6 +95,16 @@ export class TargetLocatorSync {
     exec(this.targetLocator.window(nameOrHandle));
   }
 
+  frame(nameOrElement: ElementFinderSync | string): void {
```

Comment:
Should this accept a `Locator | string` instead so the caller has more flexibility in how they select the frame?